proto/nodeinstance/info/v1/info.proto

```diff
@@ -31,6 +31,9 @@ message UpdateNodeInfo {
   string machine_guid = 5;
 
   bool child = 6;
+
+  // true when child == true && parent runs ML for child
```

Comment:
Is child actually populated when this message is sent? We're not parsing or using from the cloud side and it's marked as deprecated: https://app.zenhub.com/workspaces/product-5ea1534e9ff5df7daace00d5/issues/netdata/product/2150

Reply:
Hi! Yes, it is populated by the agent.

So even if you don't use it, you can infer from `node_id` that this is a child, right?